pkg/util/transport_util_test.go

```diff
@@ -92,15 +100,56 @@ func Test_makeTransport(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "RegistriesClientCertificates set for registry",
+			opts: config.RegistryOptions{RegistriesClientCertificates: map[string]string{registryName: "/path/to/client/certificate.cert,/path/to/client/key.key"}},
+			check: func(config *tls.Config, pool *mockedCertPool) {
+				if len(config.Certificates) != 1 {
+					t.Errorf("makeTransport().RegistriesClientCertificates not loaded for desired registry")
+				}
+			},
+		},
+		{
+			name: "RegistriesClientCertificates set for another registry",
+			opts: config.RegistryOptions{RegistriesClientCertificates: map[string]string{fmt.Sprintf("other.%s=", registryName): "/path/to/the/certificate.cert"}},
```

Comment:
A valid `"cert,key"` pair should be provided to isolate the non-matching registry name.
```suggestion
			opts: config.RegistryOptions{RegistriesClientCertificates: map[string]string{fmt.Sprintf("other.%s", registryName): "/path/to/client/certificate.cert,/path/to/client/key.key"}},
```

Reply:
Thanks, another good catch. This didn't fail the tests because this was just testing that the certs get passed to the specified registry (and not all or an incorrect registry), so it didn't really matter what was in here.

However if someone was reading the code they'd have seen it incorrect and it would have certainly raised questions, so this is a very good catch, thanks for the review!